data/json/items/book/cooking.json

```diff
@@ -441,5 +441,62 @@
     "intelligence": 12,
     "time": "35 m",
     "fun": -1
+  },
+  {
+    "id": "cookbook_liverforkids",
+    "type": "BOOK",
+    "name": { "str": "Liver-Licious Recipes Your Kids Will Love", "str_pl": "copies of Liver-Licious Recipes" },
+    "description": "This cookbook is written by liver enthusiast Tamsyn Beckerleg.  Inside you'll find a great deal of anecdotes about the near-certain ruination of children who refuse to eat their liver.",
+    "weight": "600 g",
+    "volume": "800 ml",
+    "price": 1999,
+    "price_postapoc": 3000,
+    "bashing": 2,
+    "material": [ "paper" ],
+    "symbol": "?",
+    "color": "blue",
+    "skill": "cooking",
+    "required_level": 1,
+    "max_level": 2,
+    "intelligence": 5,
+    "time": "45 m"
+  },
+  {
+    "id": "cookbook_daintydishes",
+    "type": "BOOK",
+    "name": { "str": "Dainty Dishes Fit for a King", "str_pl": "copies of Dainty Dishes" },
+    "description": "This cookbook contains many fanciful recipes. Some of them are even worth the trouble to actually prepare. Lovely pen and ink illustrations of overweight knights and gluttonous monks line the pages.",
```

Comment:
` "description": "This cookbook contains many fanciful recipes.  Some of them are even worth the trouble to actually prepare.  Lovely pen and ink illustrations of overweight knights and gluttonous monks line the pages.",`

Reply:
Isn't that what I wrote?